web/cypress/integration/route/online-debug.spec.js

```diff
@@ -21,19 +21,20 @@ import defaultSettings from '../../../config/defaultSettings';
 
 context('Online debug', () => {
   const urisWithSpecialChars = [
-    '/get',
-    '/deps-get',
-    '/deps.get',
-    '/deps_get',
-    '/get?search=1',
-    '/get?search=%U%E',
-    '/get?search=v1&number=-1',
-    '/get?search=1+1',
+    'localhost:9000/get',
+    '127.0.0.1:9000/get',
+    'baidu.com/get',
```

Comment:
`baidu` may not exist here, could we use a library to do this check?

Reply:
> `baidu` may not exist here, could we use a library to do this check?

It will take a little time to research, let me have a try

Reply:
for data: `baidu.com/get` just a case to test the host which does not contain `www`, and network does not affect the execution of test case, so I keep `baidu.com/get` in the data ^_^ @juzhiyuan 

and for the regexp,  I just tried a simpler one to cover complex situations.